src/reducers/InputOutput.js

```diff
@@ -45,6 +45,7 @@ export function materialsRemove(state, action) {
     });
 }
 
+// TODO: Do we want to also update basis for materials export?
```

Comment:
This is a question? If so I can implement it in the same fashion as the code for the web-app saveMaterials reducer.